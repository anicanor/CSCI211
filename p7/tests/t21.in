echo t21: Insert days of week into tree, then call balanced, rebalance, balanced
insert Saturday
insert Friday
insert Tuesday
insert Monday
insert Thursday
insert Wednesday
insert Sunday
balanced
echo calling rebalance
rebalance
balanced
