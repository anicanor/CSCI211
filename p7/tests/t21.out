t21: Insert days of week into tree, then call balanced, rebalance, balanced
Tree is balanced.
calling rebalance
Tree is balanced.
